easybuild/easyconfigs/m/Miller/Miller-6.4.0.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'GoPackage'
+
+name = 'Miller'
+version = '6.4.0'
+
+homepage = 'https://miller.readthedocs.io/'
+description = """Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON"""
```

```suggestion
description = "Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON"
```
(CI style guide will complain otherwise)